packages/kit/src/runtime/client/client.js

```diff
@@ -207,7 +207,13 @@ let pending_invalidate;
  * @param {HTMLElement} _target
  * @param {Parameters<typeof _hydrate>[1]} [hydrate]
  */
-export async function create_client(_app, _target, hydrate) {
+export async function start(_app, _target, hydrate) {
```

Comment:
`target` is referenced inside `navigate`. it needs to be declared at the top level, which means the argument needs to `_target` rather than `target` otherwise it'll just be shadowing it